doc/source/cluster/kubernetes/user-guides/logging.md

```diff
@@ -134,6 +138,215 @@ Examine the FluentBit sidecar's STDOUT to see logs for Ray's component processes
 kubectl logs raycluster-complete-logs-head-xxxxx -c fluentbit
 ```
 
+(kuberay-fluentbit-daemonset)=
+## Setting up logging DaemonSet with Fluent Bit
+
+In this section, we give a walkthrough of how to set up log-emitting [Fluent Bit][FluentBit] DaemonSet for Ray pods.
+The following tutorial will be based on AWS EKS, but the same steps can be applied to other Kubernetes distributions.
+
+### Setup for AWS EKS
+
+Create an EKS cluster named `fluent-bit-demo`, as shown in the [EKS getting started guide](https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html).
+
+Create a policy file called `eks-fluent-bit-daemonset-policy.json` with the following content:
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "firehose:PutRecordBatch"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "logs:PutLogEvents",
+            "Resource": "arn:aws:logs:*:*:log-group:*:*:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "logs:CreateLogStream",
+                "logs:DescribeLogStreams",
+                "logs:PutLogEvents"
+            ],
+            "Resource": "arn:aws:logs:*:*:log-group:*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": "logs:CreateLogGroup",
+            "Resource": "*"
+        }
+    ]
+}
+```
+
+To attach this policy file to the EKS on EC2 worker nodes, execute the following sequence:
+
+```bash
+$ STACK_NAME=$(eksctl get nodegroup --cluster fluent-bit-demo -o json | jq -r '.[].StackName')
+
+$ INSTANCE_PROFILE_ARN=$(aws cloudformation describe-stacks --stack-name $STACK_NAME | jq -r '.Stacks[].Outputs[] | select(.OutputKey=="InstanceProfileARN") | .OutputValue')
+
+$ ROLE_NAME=$(aws cloudformation describe-stacks --stack-name $STACK_NAME | jq -r '.Stacks[].Outputs[] | select(.OutputKey=="InstanceRoleARN") | .OutputValue' | cut -f2 -d/)
+
+$ aws iam put-role-policy \
+    --role-name $ROLE_NAME \
+    --policy-name FluentBit-DS \
+    --policy-document file://eks-fluent-bit-daemonset-policy.json
```

Comment:
```suggestion
STACK_NAME=$(eksctl get nodegroup --cluster fluent-bit-demo -o json | jq -r '.[].StackName')

INSTANCE_PROFILE_ARN=$(aws cloudformation describe-stacks --stack-name $STACK_NAME | jq -r '.Stacks[].Outputs[] | select(.OutputKey=="InstanceProfileARN") | .OutputValue')

ROLE_NAME=$(aws cloudformation describe-stacks --stack-name $STACK_NAME | jq -r '.Stacks[].Outputs[] | select(.OutputKey=="InstanceRoleARN") | .OutputValue' | cut -f2 -d/)

aws iam put-role-policy \
    --role-name $ROLE_NAME \
    --policy-name FluentBit-DS \
    --policy-document file://eks-fluent-bit-daemonset-policy.json
```

I think we need to do this so it can be copy-pasted